### PR TITLE
Minor test cleanup

### PR DESCRIPTION
**Pull request by braxtonhall**
Just some suggestions for the test file

### CODE REVIEW

File: test/SimpleCalculator.spec.ts [hunk 1]
```diff
@@ -1,52 +1,50 @@
 import SimpleCalculator, {TooLarge, TooSimple} from "../src/SimpleCalculator";
 import {testFolder} from "@ubccpsc310/folder-test";
 
-const chai = require("chai");
-const chaiAsPromised = require("chai-as-promised");
+import chai, {expect} from "chai";
+import chaiAsPromised from "chai-as-promised";
```
Comment (braxtonhall) [inline test/SimpleCalculator.spec.ts:5]: For consistency with other sample projects switched to using the "typescript way" of importing

File: test/SimpleCalculator.spec.ts [hunk 2]
```diff
@@ -1,52 +1,50 @@
...
-const expect = chai.expect;
 
 type Input = number[];
 type Output = Promise<number>;
 type Error = "TooSimple" | "TooLarge";
 
-describe("SimpleCalculator", () => {
-    describe("Add", () => {
+describe("SimpleCalculator", function () {
+    describe("Add", function () {
```
Comment (braxtonhall) [inline test/SimpleCalculator.spec.ts:14]: These functions also get the Mocha context!

File: test/SimpleCalculator.spec.ts [hunk 3]
```diff
@@ -1,52 +1,50 @@
...
 
         let simpleCalculator: SimpleCalculator;
-        beforeEach(() => {
+        beforeEach(function () {
             simpleCalculator = new SimpleCalculator();
         });
 
-        it("should add two numbers", async function() {
+        it("should add two numbers", async function () {
```
Comment (braxtonhall) [inline test/SimpleCalculator.spec.ts:21]: Sorry this whitespace was my IDE I think

File: test/SimpleCalculator.spec.ts [hunk 4]
```diff
@@ -1,52 +1,50 @@
...
-        it("should throw TooLarge if there is a number above 1000", async function() {
+        it("should throw TooLarge if there is a number above 1000", async function () {
             const result = simpleCalculator.add([1001, 1]);
 
             return expect(result).to.be.rejectedWith(TooLarge);
         });
 
         testFolder<Input, Output, Error>(
-            "Add",
+            "Add Dynamic tests",
```
Comment (braxtonhall) [inline test/SimpleCalculator.spec.ts:46]: Probably not the best name, but I saw it was getting created as
> ```
> SimpleCalculator
>     Add
>         Add
> ```

File: test/SimpleCalculator.spec.ts [hunk 5]
```diff
@@ -1,52 +1,50 @@
...
             (input: Input): Output => {
-                const simpleCalculator = new SimpleCalculator();
```
Comment (braxtonhall) [inline test/SimpleCalculator.spec.ts:49]: The `beforeEach` is already doing this step!

File: test/SimpleCalculator.spec.ts [hunk 6]
```diff
@@ -59,6 +57,7 @@ describe("SimpleCalculator", () => {
                     } else if (expected === "TooLarge") {
                         expect(actual).to.be.instanceof(TooLarge)
                     } else {
+                        // With the errorValidator, this should be unreachable
```
Comment (braxtonhall) [inline test/SimpleCalculator.spec.ts:60]: Thought it handy to be extra clear on what the different functions are doing

File: test/SimpleCalculator.spec.ts (hunk 1 above)
Comment (kekerr) [reply test/SimpleCalculator.spec.ts:5]: Ahhhh thank you!!

**Comment by kekerr** [comment]
Thanks for your feedback @braxtonhall!!

**Pull request merged**